src/main/java/org/truffleruby/interop/InteropNodes.java

```diff
@@ -620,10 +616,6 @@ public Object write(TruffleObject receiver, Object identifier, Object value,
             return foreignToRubyNode.executeConvert(foreign);
         }
 
-        protected int getCacheLimit() {
-            return getContext().getOptions().INTEROP_WRITE_CACHE;
```

Comment:
It looks like we can remove this from `tool/options.yml`and regenerate `Options` & `OptionsCatalog` with `jt build options`.

Reply:
thanks, did, and updated the PR.